runners_manager/main.py

```diff
@@ -21,15 +24,16 @@ def maintain_number_of_runner(runner_m: RunnerManager):
         time.sleep(5)
 
 
-def main(org):
-    config = [{
-        'tags': ['centos7', 'small'],
-        'flavor': 'm1.small',
-        'image': 'CentOS 7 (PVHVM)',
-        'quantity': {
-            'min': 2,
-            'max': 4,
-        },
-    }]
-    runner_m = RunnerManager(org, config)
+def main(settings: dict, args: argparse.Namespace):
+    print(args)
+    importlib.import_module(settings['python_config'])
```

Comment:
What is this? :sweat_smile:

Reply:
the function `import_module` allow the user to put a path to a module in a string and import it.
With this way we can put a path to a config file with the python module syntax and import it easily.

Reply:
And so avoid empty import for a python setup config file